api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -275,6 +275,13 @@ message ExternalProcessor {
   // backend stream lifetime. In this case, Envoy will eventually timeout the external processor stream according to this time limit.
   // The default value is 5000 milliseconds (5 seconds) if not specified.
   google.protobuf.Duration deferred_close_timeout = 19;
+
+  // [#not-implemented-hide:]
+  // Enables sending body to the side stream server before receives the header response from that server.
+  // It only works for STREAMED body processing mode. For any other body processing modes, it is ignored.
+  // Setting this flag to true is required in cases where the side stream server must wait for at least one
```

Comment:
Based on earlier conversations in this PR, is this "must wait" or "needs the ability to wait"?

Reply:
Done